canonical-web-and-design/snapcraft.io
# Split sitemap.xml

## Done

- Split sitemap 
- add sitemap index

## Issue / Card

Fixes #3237

## QA

- Pull the branch
- Run the site using the dotrun snap with `$ dotrun`
- Should be the index sitemap: http://0.0.0.0:8004/sitemap.xml
- http://0.0.0.0:8004/blog/sitemap.xml
- http://0.0.0.0:8004/store/sitemap.xml
- http://0.0.0.0:8004/docs/sitemap.xml
- http://0.0.0.0:8004/tutorials/sitemap.xml
- http://0.0.0.0:8004/sitemap-links.xml
